js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx

```diff
@@ -39,12 +68,28 @@ export const InstanceBackfills = () => {
     InstanceHealthForBackfillsQueryVariables
   >(INSTANCE_HEALTH_FOR_BACKFILLS_QUERY);
 
+  const statusFilter = useStaticSetFilter<BulkActionStatus>({
+    name: 'Status',
+    icon: 'status',
+    allValues: backfillStatusValues,
+    allowMultipleSelections: false,
+    closeOnSelect: true,
+    renderLabel: ({value}) => <div>{labelForBackfillStatus(value)}</div>,
+    getStringValue: (status) => labelForBackfillStatus(status),
+  });
+
+  const {state: statusState} = statusFilter;
+
+  const {button, activeFiltersJsx} = useFilters({filters: [statusFilter]});
+
   const {queryResult, paginationProps} = useCursorPaginatedQuery<
     InstanceBackfillsQuery,
     InstanceBackfillsQueryVariables
   >({
     query: BACKFILLS_QUERY,
-    variables: {},
+    variables: {
+      status: statusState.size > 0 ? Array.from(statusState)[0]! : undefined,
```

Comment:
Is there a reason it's limited to just a single value? We could allow for selecting multiple.

Reply:
The `partitionBackfillsOrError` field currently only allows one. We can see if users ask for the ability to filter on more than one, and if so, update the field accordingly.